mitosheet/src/components/taskpanes/Melt/MeltTaskpane.tsx

```diff
@@ -0,0 +1,214 @@
+        const columnIdsToToggle = indexes.map(index => columnIds[index]);
+
+        const newColumnIds = [...params[param_name]];
+
+        columnIdsToToggle.forEach(columnID => {
+            if (newToggle) {
+                addIfAbsent(newColumnIds, columnID);
+            } else {
+                removeIfPresent(newColumnIds, columnID);
+            }
+        })
+
+        setParams(prevParams => {
+            return {
+                ...prevParams,
+                [param_name]: newColumnIds
+            }
+        })
+    }
+
+    return (
+        <DefaultTaskpane>
+            <DefaultTaskpaneHeader 
+                header="Unpivot Dataframe"
+                setUIState={props.setUIState}           
+            />
+            <DefaultTaskpaneBody>
+                <Row justify='space-between' align='center' title='Select a dataframe to unpivot.'>
+                    <Col>
+                        <p className='text-header-3'>
+                            Dataframe
+                        </p>
+                    </Col>
+                    <Col>
+                        <Select
+                            value={props.sheetDataArray[params.sheet_index]?.dfName}
+                            onChange={(newDfName: string) => {
+                                const newSheetIndex = props.sheetDataArray.findIndex((sheetData) => {
+                                    return sheetData.dfName == newDfName;
+                                })
+                                
+                                setParams(prevParams => {
+                                    const newParams = getDefaultParams(props.sheetDataArray, newSheetIndex);
+                                    if (newParams) {
+                                        return newParams;
+                                    }
+                                    return {
+                                        ...prevParams,
+                                        sheet_index: newSheetIndex
+                                    }
+                                });
+                            }}
+                            width='medium'
+                        >
+                            {props.sheetDataArray.map(sheetData => {
+                                return (
+                                    <DropdownItem
+                                        key={sheetData.dfName}
+                                        title={sheetData.dfName}
+                                    />
+                                )
+                            })}
+                        </Select>
+                    </Col>
+                </Row>
+                <Row justify='space-between' align='center' title='Column to use as identifier variables.'>
+                    <Col>
+                        <p className='text-header-3'>
+                            ID Variables
```

Comment:
Can we add the question mark icon here to help users find the hint about what this is.